tests/python/mkl/test_mkldnn.py

```diff
@@ -471,8 +471,8 @@ def check_convolution_training(stype):
 @with_seed()
 def test_Deconvolution():
     def check_Deconvolution_training(stype):
-        for shape in [(3, 3, 10, 10)]: # testing only 2D for now
-            data_tmp = np.random.randint(256, size=shape)
```

Comment:
Was `randint(256, size=shape)` here for a reason? All other tests have `normal(-0.1, 1, size=shape)` for input values, could this be the reason why was this test flaky? I couldn't reproduce #12579 in my version of the test.